### PR TITLE
app-emulation/rkt: update rkt

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

File: app-emulation/rkt/rkt-9999.ebuild [hunk 1]
```diff
@@ -32,7 +32,7 @@ DEPEND=">=dev-lang/go-1.2
 RDEPEND=""
```
Comment (jonboulle) [inline app-emulation/rkt/rkt-9999.ebuild]: can we move to go-1.3?

Comment (crawford) [inline app-emulation/rkt/rkt-9999.ebuild]: The SDK currently uses go-1.3.3. This is just forcing the version to be _at least_ go-1.2.

Comment (marineam) [inline app-emulation/rkt/rkt-9999.ebuild]: We are actually using go 1.3 in the SDK, but if that is a strict requirement it is worth documenting in the ebuild.

File: app-emulation/rkt/rkt-9999.ebuild [hunk 2]
```diff
@@ -2,24 +2,24 @@
...
 fi
 
-# Must be in sync with stage1/rootfs/usr/cache.sh
+# Must be in sync with stage1/rootfs/usr_from_coreos/cache.sh
 IMG_RELEASE="444.5.0"
 IMG_URL="http://stable.release.core-os.net/amd64-usr/${IMG_RELEASE}/coreos_production_pxe_image.cpio.gz"
 
-DESCRIPTION="rocket"
-HOMEPAGE="https://github.com/coreos/rocket"
+DESCRIPTION="rkt"
```
Comment (marineam) [inline app-emulation/rkt/rkt-9999.ebuild]: Maybe worth putting a real description there? :-)

File: app-emulation/rkt/rkt-9999.ebuild (hunk 1 above)
Comment (crawford) [inline app-emulation/rkt/rkt-9999.ebuild]: OK, @jonboulle says 1.3 is a requirement. I'll update the ebuild.

Comment (marineam) [inline app-emulation/rkt/rkt-9999.ebuild]: Unrelated to the 1.3 question, but either add a conflicts with the old rocket name so it gets uninstalled or add an entry to coreos-overlay/profiles/updates to inform portage that the name of the installed package has changed.

File: app-emulation/rkt/rkt-9999.ebuild (hunk 2 above)
Comment (crawford) [inline app-emulation/rkt/rkt-9999.ebuild]: Done.

**Comment by marineam** [comment]
LGTM

File: app-emulation/rkt/rkt-9999.ebuild (hunk 1 above)
Comment (crawford) [inline app-emulation/rkt/rkt-9999.ebuild]: TIL. Updated.

**Pull request merged**